SpectatorView/Compositor/CopyDLL.cmd

```diff
@@ -0,0 +1,3 @@
+@echo off
+
+call "%~dp0\..\CopyDLL.cmd" %*
```

Comment:
This file may not be worth keeping for now.

Reply:
it may also be worth assessing whether all of the associated dlls can be brought into a shared *.sln compared to having them forked in different directories